tests/models/blip/test_modeling_blip.py

```diff
@@ -342,6 +342,10 @@ def test_model_from_pretrained(self):
             model = BlipTextModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
+    @unittest.skip(reason="This test class covers encoder-decoder models that the base test does not work with.")
+    def test_pt_tf_model_equivalence(self):
+        pass
```

Comment:
Needs to be rewritten then. We cannot skip this test.

Reply:
Re-enabled!